Dockerfile

```diff
@@ -0,0 +1,55 @@
+# https://docs.docker.com/samples/rails/
+FROM ruby:2.7.3
+ 
+WORKDIR  /jinglejam
+
+COPY Gemfile /jinglejam/Gemfile
+COPY Gemfile.lock /jinglejam/Gemfile.lock
+RUN gem install bundler:2.2.16
+RUN bundler install
+RUN bundle install
+
+# Replace shell with bash so we can source files
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# Set debconf to run non-interactively
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Install base dependencies
+RUN apt-get update && apt-get install -y -q --no-install-recommends \
+        apt-transport-https \
+        build-essential \
+        ca-certificates \
+        curl \
+        git \
+        libssl-dev \
+        wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV NVM_DIR /usr/local/nvm
```

Comment:
do we need a version manager for node if we are only using one version of node? we might just be making the image bigger for no reason, same goes for the base dependencies above are they all required for the application to run?

Reply:
Thats a good question, I decided to use nvm, because I was following the readme, but there may be an easier way to install node.